### PR TITLE
chore: improvements

### PR DESCRIPTION
**Pull request by KoNekoD**
Improvements:
* Makefile update(do not remove built image on down)
* Added auto recreate jwt keys when invalid jwt passphrase
* Readme improve
* Added auto-remove invalid jwt cookie
* Make errors more representative

### CODE REVIEW

File: assets/src/components/monitor/server/parts/body/processes-list/program.tsx [hunk 1]
```diff
@@ -3,11 +3,16 @@ import { FaChevronDown, FaChevronUp } from 'react-icons/fa';
...
+{
+  return !isNaN(n);
+}
+
 export const Program = ({ group, server }: { group: ApiProcessGroup; server: ApiSupervisorServer }) => {
   const list = group.processes;
   const items = list.map((p, i) => <Process process={p} server={server} key={i} />);
 
-  if (group.processes.length === 1) {
+  if (group.processes.length === 1 && !isNumeric(group.processes[0].name)) {
```
Comment (KoNekoD) [inline assets/src/components/monitor/server/parts/body/processes-list/program.tsx:15]: This code needed for this feature

image

File: src/ApiResource/Supervisor.php [hunk 2]
```diff
@@ -55,7 +55,7 @@ public static function fromArray(array $data, string $version, SupervisorServer
         }
 
         foreach ($processes as $groupName => $group) {
-            $groups[] = new ProcessGroup(name: $groupName, processes: $group);
+            $groups[] = new ProcessGroup(name: (string)$groupName, processes: $group);
```
Comment (KoNekoD) [inline src/ApiResource/Supervisor.php:58]: When process name is numeric: 0, 1, 2,... We need to force cast it to string

**Review by DevAxot — COMMENTED** [review]
everything was tested.

**Review by DevAxot — APPROVED** [review]
done!

**Pull request merged**